<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSimParameters</class>
 <widget class="QDialog" name="DialogSimParameters">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>318</width>
    <height>468</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>10000000</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Set Parameters</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>299</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="optLabel">
      <property name="text">
       <string>Optimization Scheme</string>
      </property>
      <property name="buddy">
       <cstring>optComboBox</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="utilLabel">
      <property name="text">
       <string>Utility Scheme</string>
      </property>
      <property name="buddy">
       <cstring>utilComboBox</cstring>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <layout class="QHBoxLayout" name="chainLengthLayout">
      <item>
       <widget class="QSpinBox" name="chainLengthSpinBox">
        <property name="minimum">
         <number>2</number>
        </property>
        <property name="maximum">
         <number>6</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="chainLengthSlider">
        <property name="minimum">
         <number>2</number>
        </property>
        <property name="maximum">
         <number>6</number>
        </property>
        <property name="pageStep">
         <number>2</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
        <property name="tickInterval">
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="2">
     <layout class="QHBoxLayout" name="componentSizeLayout">
      <item>
       <widget class="QSpinBox" name="componentSizeSpinBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <number>2</number>
        </property>
        <property name="maximum">
         <number>6</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="componentSizeSlider">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <number>2</number>
        </property>
        <property name="maximum">
         <number>6</number>
        </property>
        <property name="pageStep">
         <number>2</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
        <property name="tickInterval">
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QLabel" name="componentSizeLabel">
      <property name="text">
       <string>Maximum Component Size</string>
      </property>
      <property name="buddy">
       <cstring>cycleSizeSpinBox</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QComboBox" name="optComboBox">
      <item>
       <property name="text">
        <string>Utility</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Expected Utility</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Fallbacks</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Extended Fallbacks</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="chainLengthLabel">
      <property name="text">
       <string>Maximum Chain Length</string>
      </property>
      <property name="buddy">
       <cstring>cycleSizeSpinBox</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QComboBox" name="utilComboBox">
      <item>
       <property name="text">
        <string>Transplants</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5 Year Survival</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>10 Year Survival</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Score</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="cycleSizeLabel">
      <property name="text">
       <string>Maximum Cycle Size</string>
      </property>
      <property name="buddy">
       <cstring>cycleSizeSpinBox</cstring>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <layout class="QHBoxLayout" name="cycleSizeLayout">
      <item>
       <widget class="QSpinBox" name="cycleSizeSpinBox">
        <property name="minimum">
         <number>2</number>
        </property>
        <property name="maximum">
         <number>6</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="cycleSizeSlider">
        <property name="minimum">
         <number>2</number>
        </property>
        <property name="maximum">
         <number>6</number>
        </property>
        <property name="pageStep">
         <number>2</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
        <property name="tickInterval">
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>430</y>
     <width>193</width>
     <height>28</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="additionalParametersTabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>301</width>
     <height>241</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="usesScrollButtons">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="numericalParametersTab">
    <attribute name="title">
     <string>Numerical Parameters</string>
    </attribute>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>261</width>
       <height>194</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="praCheckBox">
        <property name="text">
         <string>Add Advantage to High PRA Candidates</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="praLayout">
        <item>
         <widget class="QSpinBox" name="praCutoffSpinBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="singleStep">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="praCutoffLabel">
          <property name="text">
           <string>Cutoff</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="praAdvantageSpinBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <double>10.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="praAdvantageLabel">
          <property name="text">
           <string>Value</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>18</width>
          <height>18</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="solutionsLayout">
        <item>
         <widget class="QLabel" name="solutionsLabel">
          <property name="text">
           <string>Number of Solutions</string>
          </property>
          <property name="buddy">
           <cstring>solutionsSpinBox</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="solutionsSpinBox">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>25</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="solutionsSlider">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>25</number>
          </property>
          <property name="pageStep">
           <number>2</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="estimateEUCheckBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Estimate Expected Utility</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="estimateEULayout">
        <item>
         <widget class="QSpinBox" name="numberOfEUSimSpinBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimum">
           <number>100</number>
          </property>
          <property name="maximum">
           <number>100000</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="numberOfEUSimLabel">
          <property name="text">
           <string>Number of Expected Utility Simulations</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="additionalOptionsTab">
    <attribute name="title">
     <string>Additional Options</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>259</width>
       <height>161</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="additionalOptionsLayout">
      <item>
       <layout class="QHBoxLayout" name="chainStorageLayout">
        <item>
         <widget class="QLabel" name="chainStorageLabel">
          <property name="text">
           <string>Chains Are Stored:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="chainStorageComboBox">
          <item>
           <property name="text">
            <string>As They Are Found</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>First</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Last</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="reserveOtoOBox">
        <property name="text">
         <string>Reserve O Donors for O Recipients</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkAdditionalHLABox">
        <property name="text">
         <string>Check Additional HLA Information</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="compatibleBox">
        <property name="text">
         <string>Include Compatible Pairs in Match Run</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="excludeABDonorsBox">
        <property name="text">
         <string>Exclude AB Donors from Simulation</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="allowABBridgeBox">
        <property name="text">
         <string>Allow AB Bridge Donors</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>cycleSizeSpinBox</tabstop>
  <tabstop>cycleSizeSlider</tabstop>
  <tabstop>utilComboBox</tabstop>
  <tabstop>optComboBox</tabstop>
  <tabstop>praCheckBox</tabstop>
  <tabstop>reserveOtoOBox</tabstop>
  <tabstop>checkAdditionalHLABox</tabstop>
  <tabstop>compatibleBox</tabstop>
  <tabstop>allowABBridgeBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogSimParameters</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>39</x>
     <y>592</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogSimParameters</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>39</x>
     <y>592</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cycleSizeSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>cycleSizeSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>186</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cycleSizeSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>cycleSizeSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>237</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>solutionsSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>solutionsSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>477</y>
    </hint>
    <hint type="destinationlabel">
     <x>441</x>
     <y>477</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>solutionsSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>solutionsSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>441</x>
     <y>477</y>
    </hint>
    <hint type="destinationlabel">
     <x>64</x>
     <y>477</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>excludeABDonorsBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>DialogSimParameters</receiver>
   <slot>enableBridgeDonorOptions(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>103</x>
     <y>347</y>
    </hint>
    <hint type="destinationlabel">
     <x>38</x>
     <y>358</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>praCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>DialogSimParameters</receiver>
   <slot>enablePRAOptions(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>33</x>
     <y>387</y>
    </hint>
    <hint type="destinationlabel">
     <x>71</x>
     <y>363</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chainLengthSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>chainLengthSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>195</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chainLengthSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>chainLengthSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>componentSizeSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>componentSizeSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel">
     <x>253</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>componentSizeSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>componentSizeSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>311</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>optComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>DialogSimParameters</receiver>
   <slot>enableComponentOptions(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel">
     <x>116</x>
     <y>35</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>enablePRAOptions(bool)</slot>
  <slot>enableBridgeDonorOptions(bool)</slot>
  <slot>enableComponentOptions(int)</slot>
 </slots>
</ui>
